numpy/lib/function_base.py

```diff
@@ -1537,10 +1537,10 @@ def unwrap(p, discont=pi, axis=-1):
     slice1 = [slice(None, None)]*nd     # full slices
     slice1[axis] = slice(1, None)
     slice1 = tuple(slice1)
-    ddmod = mod(dd + pi, 2*pi) - pi
-    _nx.copyto(ddmod, pi, where=(ddmod == -pi) & (dd > 0))
+    ddmod = mod(dd + discont, 2 * discont) - discont
```

Comment:
This `2*pi` change is definitely incorrect - the docs refer specifically to comparing discont to `pi` and `2*pi`, but after this change the function does not mention pi at all.

Reply:
Yes, absolutely true, the meaning of `discont` was misused here. But the logic is correct. I will correct this. I would propose a name like ´interval_size` since `min_val` and `max_val` from #14877 don't make sense at all. It only corrects differences, so it will do the same correction for 
```py
unwrap([ 1, 2, 3, 4, 5, 6, 1, 2, 3], min_val=1, max_val=7)
```
as for
```py
unwrap([ 1, 2, 3, 4, 5, 6, 1, 2, 3], min_val=0, max_val=6)
```
. Only the difference `max_val - min_val` matters.

Reply:
> since `min_val`and`max_val` from #14877 don't make sense at all.

I don't agree with that - they makes sense to me, and should satisfy `unwrap(arr, min=-a, max=a) + b== unwrap(arr + b, min=-a + b, max=a + b) `